src/python/pants/help/help_printer.py

```diff
@@ -125,67 +106,73 @@ def _print_options_help(self) -> None:
-            print(
-                f"  {self.bin_name} help [goal]                                Get help for a goal."
-            )
-            print(
-                f"  {self.bin_name} help-advanced                              Get help for global advanced options."
-            )
-            print(
-                f"  {self.bin_name} help-advanced [goal]                       Get help for a goal's advanced options."
-            )
-            print(
-                f"  {self.bin_name} help-all                                   Get help for all goals."
-            )
-            print(
-                f"  {self.bin_name} goals                                      List all installed goals."
-            )
-            print("")
-            print("  [file] can be:")
-            print("     A path to a file.")
-            print("     A path glob, such as '**/*.ext', in quotes to prevent shell expansion.")
-            print("  [target] accepts two special forms:")
-            print("    dir:  to include all targets in the specified directory.")
-            print("    dir:: to include all targets found recursively under the directory.")
-            print("\nFriendly docs:\n  http://pantsbuild.org/")
-
-            print(self._format_help(ScopeInfo(GLOBAL_SCOPE), help_request.advanced))
-
-    def _format_help(self, scope_info: ScopeInfo, show_advanced_and_deprecated: bool) -> str:
+            self._print_global_help(help_request.advanced)
+
+    def _print_global_help(self, advanced: bool):
+        print(pants_release())
+        print("\nUsage:")
+        print(
+            f"  {self._bin_name} [option ...] [goal ...] [target/file ...]  Attempt the specified goals."
+        )
+        print(f"  {self._bin_name} help                                       Get help.")
+        print(
+            f"  {self._bin_name} help [goal/subsystem]                      Get help for a goal or subsystem."
+        )
+        print(
+            f"  {self._bin_name} help-advanced                              Get help for global advanced options."
+        )
+        print(
+            f"  {self._bin_name} help-advanced [goal/subsystem]             Get help for a goal's or subsystem's advanced option."
+        )
+        print(
+            f"  {self._bin_name} help-all                                   Get help for all goals."
+        )
+        print(
+            f"  {self._bin_name} goals                                      List all installed goals."
+        )
+        print("")
+        print("  [file] can be:")
+        print("     A path to a file.")
+        print("     A path glob, such as '**/*.ext', in quotes to prevent shell expansion.")
+        print("  [target] accepts two special forms:")
+        print("    dir:  to include all targets in the specified directory.")
+        print("    dir:: to include all targets found recursively under the directory.")
+        print("\nFriendly docs:\n  http://pantsbuild.org/")
```

Comment:
It'd be nice to use dedent and a multiline string, but that's tedious to fix so fine to leave.